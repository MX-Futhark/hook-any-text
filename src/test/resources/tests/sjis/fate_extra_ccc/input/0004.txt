82BB82A482A282A682CE8163816300310000000000010000000200000003000000040000000500000006000004000000C014C90800007041000084420000803F0000803F0000000000002000230000002800000080000000FFFFFFFF0002000000010000FFFFFFFF0F000000180000003414C908DC13C9082838A34800000000000000008DA193FA82A982E795978B4982CC979082EA82F08EE682E892F782DC82E9906791CC8C9F8DB8816381630A234330303032343632303495978B498BAD89BB8C8E8AD4234344454682BE82C182BD8B4382AA82B782E981420000000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C0000001D0000001E0000001F00000020000000210000002200000B0000000000000000000000000000000000000000001F01000000000000000000000000FFFFFFFF000066430000A642000000000000604100000000FFFFFFFF000074430000A642000000000000000000006041FFFFFFFF000066430000C242000000000000604100006041FFFFFFFF000074430000C242000000008846A348020000006015C908000000000000000000000000FFFFFFFF000000005045A348020000008015C908000000000000000000000000FFFFFFFF00010000B845A34802000000A015C908000000000000000000000000FFFFFFFF000100002046A34802000000C015C908000000000000000000000000FFFFFFFF000100008846A3480200000000000000000000000000000000000000FFFFFFFF00010000F046A348020000000016C90800000000000000000000000003000100000000005847A348020000002016C9080000000000000000000000000300010000000000C047A348020000004016C908