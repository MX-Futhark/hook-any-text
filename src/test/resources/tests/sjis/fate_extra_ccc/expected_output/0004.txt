["そういえば……",
"今日から風紀の乱れを取り締まる身体検査……
風紀強化月間だった気がする。"]